.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   checks:
     docker:
-      - image: 'node:latest'
+      - image: "node:latest"
```

Comment:
Please don't change unrelated things.

Reply:
I didn't. This was the linter.

Reply:
Well probably prettier until I turned it off for this workspace.